src/platform/plugins/private/image_embeddable/public/types.ts

```diff
@@ -7,26 +7,33 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-export interface ImageConfig {
-  src: ImageFileSrc | ImageUrlSrc;
-  altText?: string;
-  sizing: {
-    objectFit: `fill` | `contain` | `cover` | `none`;
-  };
-  backgroundColor?: string;
-}
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
 
-export interface ImageFileSrc {
-  type: 'file';
-  fileId: string;
-  fileImageMeta: {
-    blurHash?: string;
-    width: number;
-    height: number;
+import type { HasDynamicActions } from '@kbn/embeddable-enhanced-plugin/public';
+import type { DynamicActionsSerializedState } from '@kbn/embeddable-enhanced-plugin/public';
+import type { DefaultEmbeddableApi } from '@kbn/embeddable-plugin/public';
+import type {
+  HasEditCapabilities,
+  HasSupportedTriggers,
+  SerializedTitles,
+} from '@kbn/presentation-publishing';
+import type { ImageConfig } from '../server';
+
+export type ImageEmbeddableSerializedState = SerializedTitles &
```

Comment:
Shouldn't `ImageEmbeddableSerializedState` just be type of `imageEmbeddableSchema`?

Reply:
https://github.com/elastic/kibana/pull/240926/commits/329085366d8a2e0ca756f32b9761564290ab5aab